javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/resolution/LambdaResolutionTest.java

```diff
@@ -187,7 +187,7 @@ void typeOfVoidLambda() {
 
         JavaParserFacade javaParserFacade = JavaParserFacade.get(new ReflectionTypeSolver());
         ResolvedType type = javaParserFacade.getType(lambdaExpr);
-        assertEquals("void", type.describe());
+        assertEquals("java.util.function.Consumer<? super java.lang.String>", type.describe());
```

Comment:
I am confused about the sample used within this test:

```java
import java.util.List;
import java.util.stream.IntStream;

public class Agenda {

    private List<String> persons;

    public void lambdaEmpty() {
        return persons.stream().forEach(p -> {
            System.out.println(p);
        });
    }

}
```

Note that `.forEach()` is void return type, AND `lambdaEmpty` is void return type... So why is there a `return ...`?

Which means that the original test is "wrong" or otherwise needs amending..?

Reply:
I add the above as a comment -- it doesn't change this PR :+1: 

Note this equivalent code where the parameter to `stringConsumer` is extracted as a variable with an explicit type:

```java
    public void lambdaEmpty() {
        Consumer<String> stringConsumer = p -> {
            System.out.println(p);
        };
        persons.stream().forEach(stringConsumer);
    }
```